platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBookUtils.java

```diff
@@ -29,6 +29,13 @@
 
 /**
  * A utility class for AddressBook functionality.
+ * <p>
+ * Format for the config.txt address book: keywork "address" followed by a comma, followed by the node ID, followed by a
```

Comment:
I think it would be more helpful to show the format something like this

`address,<nodeId>,<nickname>...`

Reply:
I am also ok with this strategy

Reply:
makes sense.